packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -136,7 +136,9 @@ class _SummaryCardState extends State<SummaryCard> {
               child: Center(
                 child: Text(
                   AppLocalizations.of(context)!.tab_for_more,
-                  style: Theme.of(context).primaryTextTheme.bodyText1,
+                  style: Theme.of(context).primaryTextTheme.bodyText1?.copyWith(
```

Comment:
What about `.textTheme.bodyText1` ?

Reply:
@VaiTon  thought of keeping the blue color so as to make that look like hyperlink to make it obvious that it can be clicked